package.json

```diff
@@ -77,6 +77,7 @@
     "chai": "4.1.2",
     "chai-as-promised": "7.1.1",
     "cucumber": "3.0.6",
+    "gulp-changed": "3.1.0",
```

Comment:
I accidentally duplicated this here in `devDependencies` by running `npm install --save-dev gulp-changed` when it's already in `dependencies`. Not sure how to fix with the lockfile system. Also, I know I've asked this before, so I'm sorry, but why are all our Gulp dependencies not `devDependencies`?

Reply:
The short answer is that they are used to create the build on Jenkins.

Reply:
Jenkins runs tests, too, and all those things are in `devDependencies`.